examples/EJS/nodejs/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "CurrentEJS",
```

Comment:
Packages are generally lowercase and dash-separated. So, `current-ejs` or `current-demo-renderer-ejs` or `c5t-demo-renderer-ejs`.